checks/evaluation/security_policy.go

```diff
@@ -15,10 +15,59 @@
 package evaluation
 
 import (
+	"fmt"
+
 	"github.com/ossf/scorecard/v4/checker"
 	sce "github.com/ossf/scorecard/v4/errors"
 )
 
+func scoreSecurityCriteria(contentLen, linkedContentLen, urls, emails, discvuls int) (int, string) {
+	score := 0
+	reason := ""
+
+	// #1: found one linked (email/http) content: score += 3
+	//     rationale: someone to collaborate with or link to
+	//     information (strong for community)
+	if urls >= 1 || emails >= 1 {
+		score += 3
+		reason += "linked content, "
+	}
+
+	// #2: more than one unique (email/http) linked content found: score += 3
```

Comment:
Is it really necessary to differentiate between 1 vs multiple links? Why would one email address not be enough?

Reply:
So - this is where I am happy to "do anything"... in my mind (hence the _rational_ in the code comments and this PR) is that more linked content is more (perhaps) helpful or insightful for the contributor wishing to communicate vuls and be informed of the disclosure process.

As such I valued more linked content (say over number of words (symbols (think groups of [:print:] separated by [:space:])--which in hindsight would make `SecurityPolicyFileInformation` really messy)

After that, then (again) without counting words, I considered the overall content length of the text file as being more than the sum of the length of the links (emails/urls) as an hint/indicator of useful content. The final plus up, is that within that texts I can find hints of numbers and words which look like disclosure and/or vulnerability.

So - in the end, the intent was to have a fairly nice gradation for the scoring over 0 or 10 (now I am just reminded I need a unit-test for only text with no hits and text which will score only 1).

So... I am open to suggestions.

Reply:
> As such I valued more linked content (say over number of words (symbols (think groups of [:print:] separated by [:space:])--which in hindsight would make `SecurityPolicyFileInformation` really messy)

One link / email address is probably enough, I'd say. This way we err on the side of caution to avoid false negatives. Wdut?

Btw, what do links typically point to in the policy file? A webpage to submit a vuln instead of an email address?

> After that, then (again) without counting words, I considered the overall content length of the text file as being more than the sum of the length of the links (emails/urls) as an hint/indicator of useful content. The final plus up, is that within that texts I can find hints of numbers and words which look like disclosure and/or vulnerability.

That sounds reasonable to me. It's similar to pre-submits that look for good description for PRs.

Reply:
> One link / email address is probably enough, I'd say. This way we err on the side of caution to > avoid false negatives. Wdut?
>
> Btw, what do links typically point to in the policy file? A webpage to submit a vuln instead of an email address?

I will think more about that - let me work these comments/suggestions, with the intent of credit for 1 url or 1 email to get a larger plus up on score.

As for what I've see, for example, Kubernetes has NO emails but numerous links, to a) point user where to read about vuls; b) how to report; and c) support-kinda-things. ossf/scorecard only has one email (NO urls) but the disclosure/vulnerability highlights are right there!

Reply:
At this point the security policy scoring is now:

```
1: one or more unique (email/http) linked content found: score += 6

2: content in the file that is more than the sum of all the links found in the first test score += 3

3: found whole number(s) and or match(es) to "Disclos" and or "Vuln" (hints to disclosure or vulnerability reporting process) score += 1
```

I think that'll suffice for a good baseline